.github/workflows/manual-package-release.yml

```
@@ -68,27 +68,50 @@ jobs:
-      # - name: Publish to crates.io
-      #   run: nix develop -c cargo release --no-confirm --execute --no-tag -p rain_orderbook_common  ${{ inputs.version-level }}
-      #   env:
-      #     CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-
-      # set npm version to rust crate version
+      
+      # set npm version
       - name: Set Version
-        run: echo "NEW_VERSION=$(node ./scripts/version.js ${{ inputs.version-level }})" >> $GITHUB_ENV
-        working-directory: packages/orderbook
+        run: |
```

actionlint (1.7.4)

property "level" is not defined in object type {version-level: string}

(expression)